docs/articles/documentation/test-api/handling-native-dialogs.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: docs
+title: Handling Native Dialogs
+permalink: /documentation/test-api/handling-native-dialogs.html
+---
+# Handling Native Dialogs
+
+TestCafe allows you to handle native browser dialogs whenever they are invoked during test run.
+You can close alert and beforeunload dialogs, choose either option in confirm dialogs or provide text to type into prompt dialogs.
```

Comment:
It would be nice to make dialog names links to [MDN](https://developer.mozilla.org/)